src/metadata/parameterGenerator.ts

```diff
@@ -106,7 +110,7 @@ export class ParameterGenerator {
         return {
             description: this.getParameterDescription(parameter),
             in: 'formData',
-            name: getDecoratorTextValue(this.parameter, ident => ident.text === 'FilesParam') || parameterName,
+            name: getDecoratorTextValue(this.parameter, ident => ident.text === 'FilesParam' || ident.text === 'StreamFilesParam') || parameterName,=======
```

Comment:
@tchambard looks like this line has a bunch of ======= signs added to the end of it. I assume by accident?

Reply:
That's what is probably breaking the CI tests

Reply:
Thanks @tnrich I fixed it.
Indeed it was a mitake...